packages/gatsby-theme-apollo-docs/src/components/page-content.js

```diff
@@ -131,6 +133,26 @@ function AsideLink(props) {
   );
 }
 
+function FeedbackLink(props) {
+  function handleClick(e) {
+    e.preventDefault();
+
+    if (window.freddyWidget) {
+      freddyWidget.show({
+        custom_fields: {
+          title: props.title
+        }
+      });
+    }
+  }
+
+  return (
+    <AsideLinkWrapper>
+      <AsideLinkInner onClick={handleClick} {...props} />
```

Comment:
Following my other comment, this could become:

```jsx
<AsideLinkWrapper>
  <AsideLinkInner onClick={handleClick}>
    <IconStar style={{marginTop: -2}} /> Rate article
  </AsideLinkInner>
</AsideLinkWrapper>
```